finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/netty4/Netty4StreamTransport.scala

```diff
@@ -551,7 +568,7 @@ private[h2] trait Netty4StreamTransport[SendMsg <: Message, RecvMsg <: Message]
     p
   }
 
-  private[this] val writeFrame: Frame => Future[Unit] = { frame =>
+  private[this] def writeFrame(frame: Frame): Future[Unit] =
```

Comment:
I changed this for consistency with the `writeHeaders` and `writeStream` defs defined above. I can revert this, but I might also convert those other defs to vals.